app/controllers/admins/calculators_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Admins
   class CalculatorsController < ApplicationController
-    before_action :find_calculator, except: %i[new create]
+    before_action :find_calculator, only: %i[show edit update]
```

Comment:
@aleksasha-o  please, rename `find_calculator`. Better to describe what it returns, then what it does.
so would be better to use `calculator`

Reply:
could you add `show`. `edit` etc missed action, please? It's more clear if we can see here what we have. 
```
def show; end
```

Reply:
SonarCloud considers this implementation a mistake. I wrote it the way it asked: https://sonarcloud.io/project/issues?id=ita-social-projects_ZeroWaste&open=AX16qDdSWNL70F3WrkKt&pullRequest=107&resolved=false&types=CODE_SMELL